test/integration/targets/egg-info/meta/main.yml

```diff
@@ -0,0 +1,2 @@
+dependencies:
```

Comment:
If we're going to keep this test (I'm not sure that we actually need to), then the installation of setuptools needs to be done in a `runme.sh` with `source virtualenv.sh` so we're not modifying the host environment.

Reply:
That's my preference at this point, unless you can think of a reason why we should maintain it.